packages/browser/src/sdk.ts

```diff
@@ -93,6 +98,11 @@ export function init(options: BrowserOptions = {}): void {
     options.defaultIntegrations = defaultIntegrations;
   }
   if (options.release === undefined) {
+    // This allows build tooling to find-and-replace __SENTRY_RELEASE__ to inject a release value
+    if (typeof __SENTRY_RELEASE__ === 'string') {
+      options.release = __SENTRY_RELEASE__;
```

Comment:
Potential pitfall of this approach is that any new release on user side will be changing vendor module source even if sentry version does not change. This will lead to new hash of vendor chunk of user app (which is usually the heaviest) and cache busting for end-users, need to download it again even if release contains only app changes.

Reply:
Hi, good point! This isn't used yet so we can change it at any point. Do you have suggestions on how to solve it differently?

Reply:
Simplest way is to provide release name as parameter at runtime.

Another way is to provide separate entry point that user need to include into build pipeline manually (or via bundler plugin, sentry webpack plugin does something similar). This entry file can have inline release name but user can choose chunk for this entry, for example inline chunk that will be inlined into `index.html`

Reply:
The SDK already has a way of configuring a release name during runtime: `Sentry.init({ release: 'your-release-name'})`

As for the separate entry point - this is something users can pretty easily do themselves. The SDK will currently already read the release name from `window.SENTRY_RELEASE.id` or `global.SENTRY_RELEASE.id` if it is set. This is the global variable that the webpack plugin and our other bundler plugins leverage to inject a release https://github.com/getsentry/sentry-javascript-bundler-plugins. If you want to disable the release injection done by the plugins you can use the `releaseInjectionTargets` option.

Considering the above, I think we can confidently leave everything as-is. I like the idea of providing a separate entry point though. We will definitely consider it in the future.

Reply:
Thanks for mentioning `releaseInjectionTargets`, was not aware of this option. 

So considering this option and ability to provide release name at runtime, changes in PR can be mitigated with careful setup for those who care about long term caching.

Thank you for thoughtful reply.